extensions/ql-vscode/src/remote-queries/view/FileCodeSnippet.tsx

```diff
@@ -208,7 +208,7 @@ const FileCodeSnippet = ({
           <div key={index}>
             {message && severity && <Message
               message={message}
-              currentLineNumber={startingLine + index}
+              currentLineNumber={startingLine + index - 1}
```

Comment:
As you say, this works, however I think it'll be more intuitive if we left this line of code as is and just moved the `Message` component below the `Box` component instead.

Reply:
That's far too sensible 😂 

(done)